pom.xml

```diff
@@ -621,6 +621,11 @@
         <version>${fasterxml.jackson.version}</version>
       </dependency>
       <dependency>
+        <groupId>com.fasterxml.jackson.dataformat</groupId>
+        <artifactId>jackson-dataformat-cbor</artifactId>
```

Comment:
what drives this -- is it just another Jackson component whose version has to be harmonized?